interfaces/builtin/browser_support.go

```diff
@@ -51,6 +51,9 @@ owner /{dev,run}/shm/{,.}com.google.Chrome.* rw,
 deny dbus (send)
     bus=session
     interface="org.gnome.GConf.Server",
+
+# Webkit uses a particular SHM names # LP: 1578217
+owner /{dev,run}/shm/WK2SharedMemory.* rw,
```

Comment:
This rule breaks application isolation and allows snaps to access files following the name /run/shm/WK2SharedMemory.*. If those are not opened with shm_open() then other snaps can interfere with the snap that opened the file.

This issue was discussed in extensively in the past. Please see my comments in https://bugs.launchpad.net/snappy/+bug/1578217/comments/10 which points at the plan of record from Gustavo at https://bugs.launchpad.net/snapcraft/+bug/1577514/comments/6.